types/setting.go

```diff
@@ -315,6 +318,16 @@ Notice that "kubernetes.io" is used as the key of all Kubernetes default tolerat
 		ReadOnly:    false,
 		Default:     "",
 	}
+
+	SettingDefinitionReplicaZoneSoftAntiAffinity = SettingDefinition{
+		DisplayName: "Replica Zone Soft Anti-Affinity",
```

Comment:
@yasker ,

Do you think we should check the original "SettingNameReplicaSoftAntiAffinity" to "SettingNameReplicaNodeSoftAntiAffinity" and "replica-soft-anti-affinity" to "replica-node-soft-anti-affinity" to make it more clear?

Thanks.

Reply:
@boknowswiki I prefer not to change the setting name, otherwise, we need to write upgrade code copy the value from the old setting to the new setting (remember it's stored in the old setting's name).